src/gluonts/dataset/artificial/recipe.py

```diff
@@ -165,16 +165,19 @@ def __rsub__(self, other):
         return LiftedSub(other, self)
 
     def __mul__(self, other):
-        return LiftedMul(self, other, operator.mul)
+        return LiftedMul(self, other)
 
     def __rmul__(self, other):
-        return LiftedMul(other, self, operator.mul)
+        return LiftedMul(other, self)
 
     def __truediv__(self, other):
-        return LiftedTruediv(self, other, operator.truediv)
+        return LiftedTruediv(self, other)
 
     def __rtruediv__(self, other):
-        return LiftedTruediv(other, self, operator.truediv)
+        return LiftedTruediv(other, self)
```

Comment:
As far as I understand, the third argument provided here is hardcoded in the specific `LiftedBinaryOp` subclass. Apparently calling the constructor with three arguments would directly invoke the parents' constructor? Not sure...

Reply:
Good catch, i would guess any of these calls would just fail.